windowsforms/Sudoku/VB/ReadMe.MD

```diff
@@ -0,0 +1,42 @@
+languages:
+    - vb
+products:
+    - dotnet-core
+    - windows
+page_type: sample
+name: ".NET Core WinForms Sudoku"
+urlFragment: "Sudoku-vb"
+description: "Sudoku is a number puzzle game, played on a nine by nine cell grid"
+# Sudoku
+## Requires
+- Visual Studio 2019 Version 16.5.0 Preview 1.0 or later
+- .Net Core 3.1
+
+## License
+- MIT
+
+## Technologies
+  - Windows Forms
+  - dotnet-core
+
+## Topics
+- Puzzle Game
+
+## Updated
+- 12/12/2019
+
+## Description
+
+<p><img id="182016" src="Sudoku-VB-2019-12-12-211405.jpg" alt="" width="274" height="252"></p>
```

Comment:
Why the usage of HTML elements here ?
There is a markdown syntax for images. Also I think there might be a better file name for the image. Maybe just `sudoko-vb.jpg` or even `sudoko.jpg` ?

Reply:
Renamed the fix, no idea how to edit Markup. The HTML is from the original file. I am guessing there was a change in for format and content between what was archived and what is now being used. Once this is accepted I will use it for the game of life. Which will be the second time I have ported it, the first time was for an Intel MDS 80 in the 1980's.